test/Serialization/Fields.jl

```diff
@@ -0,0 +1,88 @@
+@testset "Fields" begin
+    mktempdir() do path
+        @testset "UV Polynomial Over Simple Extension" begin
+            R, x = PolynomialRing(QQ, "x")
+            p = x^2 + 3//4
+            K, a = NumberField(p)
+            Ky, y = K["y"]
+            q = a * y^2 + y
+            filename = joinpath(path, "polynomial_se.uv")
+            save(q, filename)
+            loaded = load(filename)
+            L = base_ring(parent(loaded))
+            h = hom(K, L, gen(L))
```

Comment:
I believe this also works and does the same(?) thing:
```suggestion
            h = hom(K, L)
```

Reply:
the gen(L) is needed otherwise I get an failing test saying no coercion possible